libs/core/langchain_core/utils/function_calling.py

```diff
@@ -386,7 +386,7 @@ class Person(BaseModel):
                 '''Information about a person.'''
                 name: Optional[str] = Field(..., description="The name of the person")
                 hair_color: Optional[str] = Field(
-                    ..., description="The color of the peron's eyes if known"
+                    ..., description="The color of the person's eyes if known"
```

Comment:
@andyjessen, FYI:  The description refers to **_eye_** color, but the name of the attribute is **_`hair_color`_**.  The same mistake is present in the file `docs/docs/how_to/extraction_examples.ipynb`, but I couldn't add a comment to that file via the GitHub web UI.

Reply:
@lsloan - Thanks. I modified the description and submitted another PR.